aws-codepipeline-transition-disabler/.gitignore

```diff
@@ -0,0 +1,5 @@
+/target/
```

Comment:
thoughts about having .gitkeep files in the `/src/main/resources`, `/src/test/java`, and `/src/test/resources`